libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
Sure, that makes sense as a tab is a tab after all. So I'm guessing if I switch to tab indent it will at least be right some of the time...